movies-app/src/components/MovieForm/MovieForm.js

```diff
@@ -0,0 +1,99 @@
+import { useState } from 'react';
+import './MovieForm.css';
+
+const MovieForm = ({ movieInfo }) => {
+    const [formData, setFormData] = useState(movieInfo || {});
+    const handleChange = (e) => {
+        const { name, value } = e.target;
+        setFormData((prevData) => ({
+            ...prevData,
+            [name]: value,
+        }));
+    };
+
+    const handleSubmit = (e) => {
+        e.preventDefault();
+        console.log(formData);
+        // onSubmit(formData);
+        // onClose();
+    };
+
+    const resetForm = () => {
+        setFormData(movieInfo || {})
+    }
+
+    return (
+            <form onSubmit={handleSubmit}>
+                <div className="row">
+                    <div className="col-8">
+                        <div className="mb-4">
+                            <label for="title" className="form-label">Title</label>
```

Comment:
Put all the hardcoded values in a constant file

Reply:
Moved labels to constant file